### PR TITLE
criminal records cl

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
cl

## Why / Balance
evil pjbot

## Technical details
no

## Media
find out
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:trollface:

### CODE REVIEW

**Comment by deltanedas** [comment]
@AJCM-git pro

**Comment by mirrorcult** [comment]
dont add it manually to the file, make a yml entry in `Changelog/Parts`

**Review by mirrorcult — CHANGES_REQUESTED** [review]
above

**Review by AJCM-git — CHANGES_REQUESTED** [review]
So just got a call from one of our changelog engineers in floor 11 and apparently the bot would prune this and the way to manually adding changelogs is making a parts file like in  #14457 and  #21259 , who would have thought (also when you do the authors put ficcialfaint and deltanedas since i dont want something to blow up cuz some random check looks up special characters for some reason)

File: Resources/Changelog/Parts/parts_22510.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: ficcialfaint and deltanedas
+changes:
+- message: Criminal records and the console that manages them have been added to the game. They also have a guidebook entry.
+  type: Add
```
Comment (AJCM-git) [inline Resources/Changelog/Parts/parts_22510.yaml:4]:
```suggestion
- type: Add
  message: Criminal records and the console that manages them have been added to the game. They also have a guidebook entry.
```

**Review by AJCM-git — APPROVED** [review]

**Comment by deltanedas** [comment]
motherfucker @PJBot !!!!!

**Comment by AJCM-git** [comment]
this is so cursed

**Pull request merged**